src/Raven.Server/Documents/AI/AiAssistant/Responses/AiAssistantResponseStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Raven.Server.Documents.AI.AiAssistant.Responses;
+
+public enum AiAssistantResponseStatus
+{
+    Success,
+    InvalidCredentials,
+    InvalidData,
+    ConsentRequired,
+    OutOfTokens
```

Comment:
what about "internal error"

Reply:
this all actually not needed, will remove it.